---

input_file: "pcd_images/PartNet/table/table_01.pcd"
output_file: "pcd_images/PartNet/table/table_01_filtered.pcd"


save_output: true
translate_output: false
automatic_bounds: false
use_clustering: true
new_scan: false
transform_input: true

pre_rotation:    [0.0, 3.14159, 0.0] # ds435i rgb camera 32inches from table surface pointing down
pre_translation: [0.0, 0.0, 0.8128] # 32.0*0.0254 

bounding_box: [-0.5, 0.5, -0.5, 0.5, -0.25, 0.25] # bounding box limits
voxel_size: 0.01

